src/main/java/edu/umich/ctools/sectionsUtilityTool/FriendServlet.java

```diff
@@ -105,7 +105,7 @@ private void socialLoginRestApiCall(HttpServletRequest request,
             String nonUmichEmailFirstName = URLEncoder.encode(request.getParameter(PARAMETER_FRIEND_FIRST_NAME), Utils.CONSTANT_UTF_8);
             String nonUmichEmailLastName = URLEncoder.encode(request.getParameter(PARAMETER_FRIEND_LAST_NAME), Utils.CONSTANT_UTF_8);
             String keySecretStr = String.format("%s:%s", key, secret);
-
+            M_log.info(String.format("Social login guest invitation API Request for user Email: %s Firstname: %s Lastname: %s ", nonUmichEmail, nonUmichEmailFirstName, nonUmichEmailLastName));
```

Comment:
I am logging all the parameters from requests as info at the very beginning so that I don't need to log more info with error message